JitsiConferenceEventManager.js

```diff
@@ -167,6 +167,26 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function () {
             chatRoom.eventEmitter.emit(
                 XMPPEvents.CONFERENCE_SETUP_FAILED,
                 new Error("ICE fail"));
+            conference.eventEmitter.emit(
+                JitsiConferenceEvents.ICE_STATE_CHANGED,
+                conference.myUserId(),
+                "failed");
+        });
+
+    chatRoom.addListener(XMPPEvents.CONNECTION_RESTORED,
+        () => {
+            conference.eventEmitter.emit(
```

Comment:
Hi @nbermudezs, thanks for the PR! A restored XMPP connection does not imply that ICE has been restored. Maybe the wording needs to be different, but, at first glance, this doesn't look right.